content/_index.md

```diff
@@ -0,0 +1,10 @@
+---
+title: "CytoData Society"
+---
+
+![CytoData banner](/./main_files/cytodata-banner.png)
```

Comment:
add "The field of image-based profiling is still in its infancy, and its adoption has been slow, primarily due to the data analysis challenges posed, the large imaging experiments required, and the lack of ground-truth data sets where relationships among images are known. The CytoData Society aims to bring image-based profiling into the mainstream by organizing a striving community working on this technology."